default.json

```diff
@@ -35,6 +35,10 @@
       "commitMessageSuffix": "{{baseDir}}-",
       "bumpVersion": "patch",
       "packagePatterns": [".*"]
+    },
+    {
+      "matchPackagePatterns": ["curl"],
+      "automerge": true
```

Comment:
based on https://docs.renovatebot.com/key-concepts/automerge/

Reply:
will it work this way, as package name is debian_11/curl
in https://github.com/Linkurious/linkurious-server/pull/4423